src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -257,6 +257,9 @@ private function fetchUrl($url, $locale, $limit, $region)
             if (isset($result->formatted_address)) {
                 $address = $address->withFormattedAddress($result->formatted_address);
             }
+            if (property_exists($builder, 'subpremise')) {
```

Comment:
I dynamically added 'subpremise' property to AddressBuilder class, because that class is shared among other providers and contains properties that are common for all providers. 
That's why I didn't add it as filed to the class, and it doesn't have setter..  
I'm checking if 'subpremise' exists in $builder, because if it exists in result, then it is dynamically added to $builder object.